com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviourUpdater.cs

```diff
@@ -28,20 +28,28 @@ internal void NetworkBehaviourUpdate(NetworkManager networkManager)
                         m_Touched.UnionWith(spawnedObjs);
                         foreach (var sobj in spawnedObjs)
                         {
-                            // Sync just the variables for just the objects this client sees
-                            for (int k = 0; k < sobj.ChildNetworkBehaviours.Count; k++)
+                            // Under specific conditions this can become null, so we don't want to access it if it is null
```

Comment:
Do we know these specific conditions? Are they expected behaviour / fine?

Reply:
Yeah, I think that was due to NetworkObjects being destroyed locally by the client if exiting ungracefully before the changes were put in to assert if you are the client and you try to destroy a NetworkObject.
Now it just throws an assert for every NetworkObject being destroyed by the client on an ungraceful exit...so I guess I can remove this?  (wasn't sure if we were keeping that assert that or not)